templates/security_groups.cfn.yml.j2

```diff
@@ -25,70 +25,6 @@ Parameters:
     Description: Name of the platform ie GI, Services, Static
 
 Resources:
-  CloudfrontHTTP:
```

Comment:
we can't delete till after things move (unless all cloudfront references are in this cfn, if that is the case then this can go)